config/checkstyle_checks.xml

```diff
@@ -371,6 +371,9 @@
       <property name="severity" value="ignore"/>
     </module>
     <module name="NoClone"/>
+    <module name="NoEnumTrailingComma">
+      <property name="severity" value="ignore"/>
```

Comment:
please add comment to explain why it is ignored. see example for NoArrayTrailingComma

Reply:
fixed.

Reply:
There is no such Check, we had almost completed implementation is scope of https://github.com/sevntu-checkstyle/sevntu.checkstyle/issues/464 , but PR was abandoned.